results-tabulation-api/ext/ExtendedElection/ExtendedElectionProvincialCouncilElection2021/__init__.py

```diff
@@ -207,7 +207,9 @@ def _get_party(row):
             root_election.add_party(partyId=party.partyId)
 
             if candidate_type == CANDIDATE_TYPE_NORMAL:
+                province_election = _get_province_election(row)
                 administrative_district_election = _get_administrative_district_election(row)
+                province_election.add_party(partyId=party.partyId)
```

Comment:
The candidates also should be mapped to each election as well
https://github.com/ECLK/results-tabulation/pull/846/files#diff-8146ca8ae39d50809eb01f4f470be5bd03fafd168e5c5d48aadfe684cf6df55eR182

Reply:
commit pushed